AK/Math.h

```diff
@@ -455,4 +456,20 @@ constexpr T pow(T x, T y)
 
 #undef CONSTEXPR_STATE
 
+// Check that `value` is in `[min,max]`
+template<typename T>
+constexpr bool is_in_bounds(T const& value, T const& min, T const& max)
```

Comment:
These names feel somewhat arbitrary to me, what makes `range` end-exclusive and `bounds` end-inclusive?

(I don't really understand what utility these provide over the imo much more clear boolean expressions they contain, but I don't really care enough to raise that as an actual issue)